charts/lakefs/templates/configmap.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "lakefs.fullname" . }}
 {{- with .Values.lakefsConfig }}
 data:
-  config.yaml:
-    {{- toYaml . | nindent 2 }}
+  config.yaml: {{- toYaml . | nindent 2 }}
 {{- end }}
+{{- if .Values.secretKeys.licenseContents }}
+  license.tkn: {{ .Values.secretKeys.licenseContents }}
+{{- end }}
```

Comment:
As a best practice in Helm (we don't always follow) you should add all possible values to the `values.yaml` 
Otherwise users will need to go into the implementation to understand how to set this value. 

In this case the `values.yaml` should have: 

```yaml
secretKeys:
  licenseContents: null
```